### DISCUSSION TITLE
Creating Template For Aspire

### DISCUSSION THREAD

**Discussion by KFKMan** [microsoft/aspire#3430] [Q&A]
i want to create template for Aspire.
Type 1 => AppHost, ServiceDefaults, Core
Type 2 => Api, EntityLayer.... (all of them must be have reference to Core) (Api must be auto recognized by AppHost like Aspire WebApi template)....

How can i create a template like that?

**Comment by davidfowl** [comment]
Take a look at our starter template https://github.com/dotnet/aspire/tree/main/src/Aspire.ProjectTemplates/templates/aspire-starter

**Reply by KFKMan** [reply]
i looked at that. i want auto recognize existing AppHost and auto adding code's to that.

**Reply by davidfowl** [reply]
I don't think that's possible with a template
